src/workos.ts

```diff
@@ -127,6 +143,10 @@ export class WorkOS {
       if (response) {
         const { status, data, headers } = response;
         const requestID = headers['X-Request-ID'];
+        const err = 'error';
```

Comment:
Could we access this properties would storing the keys in new variables?

```ts
const error = data.error;
const description = data['error_description'];
```

We might even be able to write it like this:

```ts
const { error, error_description: errorDescription } = data;
```